src/raven.js

```diff
@@ -266,8 +266,15 @@ Raven.prototype = {
         }
 
         // If this has already been wrapped in the past, return that
-        if (func.__raven_wrapper__ ){
-            return func.__raven_wrapper__ ;
+        try {
+            if (func.__raven_wrapper__ ){
+                return func.__raven_wrapper__ ;
+            }
+        } catch (e) {
+            // Just accessing the __raven_wrapper__ prop in some Selenium environments
+            // can cause a "Permission denied" exception (see raven-js#495).
+            // Bail on wrapping and return the function as-is (defers to window.onerror).
+            return func;
```

Comment:
There's a try/catch block above for `__raven__` that also bails and returns `func`. Rather than repeat that code, lets move the check into that try/catch block.